src/components/AddSenior.tsx

```diff
@@ -214,124 +250,206 @@ const AddSenior = ({
         setConfirm(true);
         setSeniors([...seniors, newSeniorObj]);
       }
-    } 
+    }
     // check after both API calls
     if (currRes.status != 200) {
-      console.log(currRes.text().then((text) => { console.log(text); }));
+      console.log(
+        currRes.text().then((text) => {
+          console.log(text);
+        })
+      );
       setError(true);
     }
 
     setSeniorData(emptySenior);
     setSelectedStudents([]);
   };
 
+  const handleImageReplace = (event: React.ChangeEvent<HTMLInputElement>) => {
+    const files = event.target.files;
+
+    if (!files || files.length === 0) return;
+
+    const selectedFile = files[0];
+    if (!selectedFile) return;
+    const reader = new FileReader();
+
+    reader.onload = (loadEvent: ProgressEvent<FileReader>) => {
+      const dataUrl = loadEvent.target?.result;
+      if (typeof dataUrl === "string") {
+        setCurrentImage(dataUrl);
+      }
+    };
+
+    reader.readAsDataURL(selectedFile);
+  };
   return (
     <>
-      {showAddSeniorPopUp && 
-        (
-        <div className="absolute top-0 left-0 z-50 flex h-screen w-screen flex-row items-center justify-center \
-                        text-left backdrop-blur-[2px] backdrop-brightness-75 md:w-full">
-          
-          <div className={cn("sticky flex min-h-1/4 p-10 flex-col justify-between rounded-lg bg-white", 
-                          confirm || error ? "top-[12.5%] w-2/5" : "top-[5%] sm:w-4/5 md:w-1/2")}>
-          {!confirm && !error ? (
-                <>
+      {showAddSeniorPopUp && (
+        <div
+          className="absolute bottom-0 left-0 top-10 z-50 flex w-screen flex-row items-center justify-center
```

Comment:
This looks a bit weird on large screen

![image](https://github.com/JumboCode/TheLegacyProject2/assets/74647679/49c11f61-c4a5-44f3-9663-42c2a7574ecb)

What about using this styling instead? `top-1/2 left-1/2 transform -translate-x-1/2 -translate-y-1/2`